### ISSUE TITLE
jspsych-video not playing in Edge

### ISSUE THREAD

**Issue by breslavya** [jspsych/jsPsych#517]
When testing my experiment's browser-compatibility, I found I wasn't able to present videos in edge, even with the simplest timeline presenting a video, e.g.
```
    var timeline = [];
    var vid_trial = {
      type: 'video',
      sources: [movie.mp4'],
      width: 320,
      height: 240,
      controls: false,
      autoplay: true
    };
    timeline.push(vid_trial);
    jsPsych.init({timeline: timeline})
```

Maybe it's an issue with some settings on my browser, but I'm posting it here in case it's a more general problem with jspsych-video on edge.

It seems the problem is the addition of start time (as the code adds start-time=0 when none is specified) - commenting out the following code in my local copy of jspsych-video did the trick:
```
      if (trial.start !== null) {
        video_html+= '#t=' + trial.start;
      } else {
        video_html+= '#t=0';
      }
      if (trial.stop !== null) {
        video_html+= ',' + trial.stop
      }
```
And when simply presenting a video in html this works:
```
<video width="320" height="240" autoplay><source src="movie.mp4" type="video/mp4"></video>
```
and this doesn't:
```
<video width="320" height="240" autoplay><source src="movie.mp4#t=0" type="video/mp4"></video>
```
tested on:
Microsoft Edge 40.15063.674.0
Microsoft EdgeHTML 15.15063

Aya

**Comment by jodeleeuw** [comment]
Looks like this feature is unsupported in Edge:
https://caniuse.com/#feat=media-fragments

I'll work on a replacement solution.

**Comment by breslavya** [comment]
Great, thanks​

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by jodeleeuw** [comment]
This is working now, and will be released with 6.0.2

**Comment by breslavya** [comment]
Thanks, that was fast!​
